core/test/data_array_test.cpp

```diff
@@ -33,3 +33,92 @@ TEST(DataArrayTest, sum_dataset_columns_via_DataArray) {
   // but + does not.
   EXPECT_EQ(sum, dataset["data_zyx"]);
 }
+
+auto make_sparse() {
+  auto var = makeVariable<double>({Dim::Y, Dim::X}, {2, Dimensions::Sparse});
+  var.setUnit(units::us);
+  auto vals = var.sparseValues<double>();
+  vals[0] = {1.1, 2.2, 3.3};
+  vals[1] = {1.1, 2.2, 3.3, 5.5};
+  return DataArray(std::optional<Variable>(), {{Dim::X, var}});
+}
+
+auto make_histogram() {
+  auto edges = makeVariable<double>({{Dim::Y, 2}, {Dim::X, 3}}, units::us,
+                                    {0, 2, 4, 1, 3, 5});
+  auto weights = makeVariable<double>({Dim::X, 2}, {2.0, 3.0}, {0.3, 0.4});
```

Comment:
I this really weights in a histogram? I thought weights were for sparse data. Maybe there is a better name?

Reply:
I don't know, do you have a better suggestion, I don't mind changing it? I think it depends on what the histogram contains. If it is counts then I absolutely agree that "weights" would be a wrong name. Here it is a scale or weight(?) factor.

Reply:
Just `data` ?

Reply:
It's really not that important.... it can also stay the way it is. Was just something that confused me a little when I was reading, that the weights were in the histogram